services/ui-src/src/utils/other/export.tsx

```diff
@@ -17,7 +18,22 @@ import {
   otherSpecify,
 } from "utils";
 // verbiage
-import verbiage from "verbiage/pages/mcpar/mcpar-export";
+import McparVerbiage from "verbiage/pages/mcpar/mcpar-export";
+import MlrVerbiage from "verbiage/pages/mlr/mlr-export";
+import NaaarVerbiage from "verbiage/pages/naaar/naaar-export";
+
+export const getVerbiage = () => {
+  const reportType = localStorage.getItem("selectedReportType");
+  switch (reportType) {
+    case ReportType.MLR:
+      return MlrVerbiage;
+    case ReportType.NAAAR:
+      return NaaarVerbiage;
+    case ReportType.MCPAR:
+    default:
+      return McparVerbiage;
+  }
+};
```

Comment:
Purpose: make an easy way to switch between verbiage files so that we can have the verbiage files determine required and optional text instead of the display function.

I think this could be a nice method to make its own util and use more broadly, but I didn't want to refactor too much here.

Reply:
working on this refactor [here](https://github.com/Enterprise-CMCS/macpro-mdct-mcr/pull/12141)